src/Ocelot.Cache.CacheManager/Ocelot.Cache.CacheManager.csproj

```diff
@@ -43,18 +43,18 @@
   </ItemGroup>
   <!-- Conditionally obtain references for the net 6.0 target -->
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.1" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.1" />
     <PackageReference Include="Microsoft.NETCore.Platforms" Version="6.0.13" />
   </ItemGroup>
   <!-- Conditionally obtain references for the net 7.0 target -->
   <ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="7.0.4" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
```

Comment:
Are you sure about this? You're targeting net7.0...

Reply:
Yes, I am. Could you check this link: [Microsoft.Extensions.Configuration/8.0.0 | Frameworks](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/8.0.0#supportedframeworks-body-tab)
![image](https://github.com/user-attachments/assets/bc1ff84b-d112-4ee7-8fc1-8c305d2ff047)
So, `net7.0` is included into the list.

Reply:
The strategy is straightforward: search for the most updated packs by date that have the max version number and supporting the current target framework. Therefore:
- All .NET 7 references have updated to version `8.0.x`, targets `net7.0`
- All .NET 8 references have been updated to version `9.0.0`, which targets `net8.0`

Is it now clear? For further information please review the updated test projects where the same strategy has applied.